### ISSUE TITLE
Series K

### ISSUE THREAD

**Issue by johnx102** [kyleaa/homebridge-samsungtv2016#7]
Hi , i have a SAMSUNG UE65KS9000 , and i have some issues with home bridge , and maybe there is a thing i do not understand.
First, in home on iOS, samsungTv is show like a switch, so you can just on or off... 
Second problem, if i tried to switch Off Tv with the switch, my homebridge stop running because a problem....

Can you help me, if you need i can give log for error.
Thanks your you're help ;)

**Comment by ryck** [comment]
Well, right now it's just a switch, you can only turn it on or off, so nothing wrong there, is working as expected...

It works for me (using the home app, siri or other third party apps), so maybe you can post the logs so we can see what's going on?

**Comment by johnx102** [comment]
I retest Today and it's working ....I just only see the samsung ask me autorisation for home bridge, maybe it's that, but it's working !! Thank's a lot for you're plugin ;)

**Comment by johnx102** [comment]
and it's working in wifi too ;)

**Comment by johnx102** [comment]
sometime i have this error and my home bridge stop.
It's lookalike same error than first time events.js:160
stdout
14:04:46
      throw er; // Unhandled 'error' event
stdout
14:04:46
      ^
stdout
14:04:46
stdout
14:04:46
Error: connect ECONNREFUSED 192.168.1.33:8001
stdout
14:04:46
    at Object.exports._errnoException (util.js:1036:11)
stdout
14:04:46
    at exports._exceptionWithHostPort (util.js:1059:20)
stdout
14:04:46
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1080:14)

but maybe because it's in wifi ....
but after it's reworking... strange

**Comment by johnx102** [comment]
and now seems to reworking normally... i'm still in wifi
so i want and tell you ...

**Comment by johnx102** [comment]
i think if it' off, after too much time there is no communication ....
i will retest in ethernet

**Comment by inzlabs** [comment]
I will jump into this, sadly with a problem but no much to add on how to find a solution. I have a: 

```
  "name": "[TV] Samsung 6 Series (50)",
  "version": "2.1.0",
  "device": {
    "type": "Samsung SmartTV",
    "modelName": "UN50KU6300",
    "ip": "192.168.1.13",
    "firmwareVersion": "Unknown",
    "name": "[TV] Samsung 6 Series (50)",
    "resolution": "3840x2160",
    "countryCode": "US",
    "msfVersion": "2.1.0",
    "smartHubAgreement": "true",
    "OS": "Tizen"
```

Once this plugin installed, I can see it in the Home app as an accessory, and also on the Homebridge. It stays the TV is on. When I try to turn it off, instantaneously the TV asks me for an allow permission. I allow. But the switch off, doesn't work. 

At the same time, I can't change the location of the TV (to living room), or the name of the TV on the Home app. 

Also using another app: Koogeek Home brings this error: https://imgur.com/gallery/v3xBg

If anyone has an idea on how to fix this, I am all ears and eyes.

ps. I research all other Samsung TV plugins, this work better with 2016 tvs. Thank you for your work.

**Comment by kyleaa** [comment]
You have to change the name of the accessory in your homebridge config. I had no issues moving between rooms in the Home app. I have no clue what that other app is doing.

The issue with turning off is being tracked in github issue #11 - it may be fixed in npm release 0.0.6
